src/coreclr/jit/lclmorph.cpp

```diff
@@ -1535,14 +1534,16 @@ class LocalAddressVisitor final : public GenTreeVisitor<LocalAddressVisitor>
     //
     // Arguments:
     //    lclNum - the local number to update the count for.
+    //    node   - local node representing the reference
+    //    user   - "node"'s user
     //
     // Notes:
     //    fgMakeOutgoingStructArgCopy checks the ref counts for implicit byref params when it decides
     //    if it's legal to elide certain copies of them;
     //    fgRetypeImplicitByRefArgs checks the ref counts when it decides to undo promotions.
     //    fgForwardSub uses ref counts to decide when to forward sub.
     //
-    void UpdateEarlyRefCount(unsigned lclNum)
+    void UpdateEarlyRefCount(unsigned lclNum, GenTree* node, GenTree* user)
```

Comment:
The notes above here are stale, `fgMakeOutgoingStructArgCopy` now relies on early liveness, as does `fgForwardSub` (by default).

We can clean this up later, but if you want to tackle it now there's a bit of dead code and some stale comments over in `fgMakeOutgoingStructArgCopy` too.

Reply:
Deleted. Looks like substitution also has some stale-ish comments; left those for another day.